src/status_im/chat/subs.cljs

```diff
@@ -52,15 +52,17 @@
       platform/ios? kb-height
       :default 0)))
 
-(defn active-chats [[_ chat]]
-  ;;TODO (andrey) console should be shown in dev mode only, will be done soon
-  (and (:is-active chat))) ;(not= const/console-chat-id (:chat-id chat))))
+(defn active-chats [dev-mode?]
+  (fn [[_ chat]]
+    (and (:is-active chat)
+         (not (and (not dev-mode?) (= const/console-chat-id (:chat-id chat)))))))
```

Comment:
I think this can be rewritten more concisely as:
```
(and (:is-active chat)
     (or dev-mode?
         (not= const/console-chat-id (:chat-id chat))))
```

Reply:
no this is not the same

Reply:
we should hide console the dev-mode = false, so this should be false if this is console and dev-mode = false

Reply:
mmh, I had to work it out, but they look the same to me, yours is:

```
a Λ ¬(¬b Λ c)
```

while the one I wrote is:

```
a Λ (b V ¬c)
```

(Did you notice the  `not=` console?)

If dev-mode is not enabled:

```
is-active && (false || not-console)
```
which should simplify to

```
is-active && not-console
```

if dev is enabled then we just really check `is-active`, as anything goes

Am I missing something here?

Reply:
in your case you'll show all chats in dev-mode but we need to show only console

Reply:
If chat is `{:is-active false}` my code would return false, as it would just short circuit the `and`,  only the second line has been changed, first line the code is identical, is that what you meant?

Reply:
Or you mean to say that on dev mode we only show the console? no other active chats?

Reply:
you're right. my code looks weird, i'll change it, thanks

Reply:
no worries, thanks for being patient with the question :)